core/new-gui/src/app/workspace/component/navigation/navigation.component.html

```diff
@@ -9,22 +9,49 @@
 
     <!-- workflow metadata display -->
     <div *ngIf="userSystemEnabled" class="texera-navigation-workflow-metadata">
+      <button
+        (click)="closeParticularVersionDisplay()"
+        *ngIf="displayParticularWorkflowVersion"
+        nz-button
+        nzType="text"
+        title="back"
+        class="particular-version-metadata"
+      >
+        <i nz-icon nzTheme="outline" nzType="arrow-left"></i>
+      </button>
       <label>
         <input
+          *ngIf="!displayParticularWorkflowVersion"
           (change)="onWorkflowNameChange()"
           [(ngModel)]="currentWorkflowName"
           class="workflow-name"
           placeholder="Untitled Workflow"
         />
+        <span *ngIf="displayParticularWorkflowVersion" class="particular-version-metadata" spellcheck="false">
+          {{particularVersionDate}}
+        </span>
       </label>
-      <span class="save-state" spellcheck="false" (click)="onClickGetAllVersions()">{{autoSaveState}}</span>
+      <button *ngIf="displayParticularWorkflowVersion" class="btn btn-primary btn-sm" (click)="revertToVersion()">
```

Comment:
This button is using class `btn btn-primary btn-sm`, this is actually from bootstrap theme (not the ng zorro theme). Please change it to use ng-zorro, thanks.
https://ng.ant.design/components/button/en

Reply:
done